srt/server.go

```diff
@@ -67,31 +66,29 @@ func NewServer(config *Config) Server {
 
 // Listen sets up a SRT socket in listen mode
 func (s *ServerImpl) Listen(ctx context.Context) error {
-	host, portString, err := net.SplitHostPort(s.config.Address)
-	if err != nil {
-		return err
-	}
-
-	if len(host) == 0 {
-		host = "localhost"
-	}
-
-	port, err := strconv.ParseUint(portString, 10, 16)
-	if err != nil {
-		return err
-	}
-
-	addrs, err := net.LookupHost(host)
-	if err != nil {
-		return err
-	}
-
-	for _, address := range addrs {
-		err := s.listenAt(ctx, address, uint16(port))
+	for _, address := range s.config.Addresses {
+		host, portString, err := net.SplitHostPort(address)
+		if err != nil {
+			return err
+		}
+		if len(host) == 0 {
+			host = "localhost"
```

Comment:
I think that does not match the default behavior in go. Go binds to all addresses when the host is omitted. So I think in this case we should bind to `0.0.0.0` and `::`